google/cloud/storage/idempotency_policy_test.cc

```diff
@@ -516,6 +516,51 @@ TEST(StrictIdempotencyPolicyTest, PatchDefaultObjectAclIfMatchEtag) {
   EXPECT_TRUE(policy.IsIdempotent(request));
 }
 
+TEST(StrictIdempotencyPolicyTest, GetProjectServiceAccount) {
+  StrictIdempotencyPolicy policy;
+  internal::GetProjectServiceAccountRequest request(
+      "test-project-id");
+  EXPECT_TRUE(policy.IsIdempotent(request));
+}
+
+TEST(StrictIdempotencyPolicyTest, ListNotification) {
+  StrictIdempotencyPolicy policy;
+  internal::ListNotificationsRequest request("test-bucket-name");
+  EXPECT_TRUE(policy.IsIdempotent(request));
+}
+
+TEST(StrictIdempotencyPolicyTest, CreateNotification) {
+  StrictIdempotencyPolicy policy;
+  internal::CreateNotificationRequest request("test-bucket-name", NotificationMetadata());
+  EXPECT_FALSE(policy.IsIdempotent(request));
+}
+
+TEST(StrictIdempotencyPolicyTest, CreateNotificationIfMatchEtag) {
+  StrictIdempotencyPolicy policy;
+  internal::CreateNotificationRequest request("test-bucket-name", NotificationMetadata());
+  request.set_option(IfMatchEtag("ABC123="));
+  EXPECT_TRUE(policy.IsIdempotent(request));
+}
+
+TEST(StrictIdempotencyPolicyTest, GetNotification) {
+  StrictIdempotencyPolicy policy;
+  internal::GetNotificationRequest request("test-bucket-name", "test-notification-id");
+  EXPECT_TRUE(policy.IsIdempotent(request));
+}
+
+TEST(StrictIdempotencyPolicyTest, DeleteNotification) {
+  StrictIdempotencyPolicy policy;
+  internal::DeleteNotificationRequest request("test-bucket-name", "test-notification-id");
+  EXPECT_FALSE(policy.IsIdempotent(request));
+}
+
+TEST(StrictIdempotencyPolicyTest, DeleteNotificationIfMatchEtag) {
+  StrictIdempotencyPolicy policy;
+  internal::DeleteNotificationRequest request("test-bucket-name", "test-notification-id");
+  request.set_option(IfMatchEtag("ABC123="));
```

Comment:
Same thing here.  Etag doesn't look to have any effect on delete requests (each config has a unique ID, which makes etags unnecessary).

Reply:
Ack.  I will change this to always be idempotent then.

Reply:
Done.